ag-grid-webcomponent/index.js

```diff
@@ -123,16 +125,55 @@ class AgGrid extends HTMLElement {
     if (selectedIds.length == 0) {
       this.api.deselectAll();
     } else {
-      this.api.forEachNode(function (node) {
+      this.api.forEachNode(function(node) {
         const selected = selectedIds.includes(node.id);
         node.setSelected(selected);
       });
     }
   }
+
   set columnDefs(defs) {
     this.api.setColumnDefs(defs);
   }
 
+  set getContextMenuItems(data) {
+    const prepareContextAction = (item, params) => {
+      if (typeof item === "string") return item;
+
+      if (typeof item.actionName === "string") {
+        item.action = () => {
+          const contextMenuEvent = new CustomEvent("oncontextaction", {
```

Comment:
This event's name is a bit inconsistent with the others. There's no `on` part. In Elm, this reads a bit clunky: `Html.Events.on "oncontextaction" event`. Also, AgGrid events are like `cellValueChanged` or `gridReady` too.

Also, the other custom events were using camelCase. We might wanna stick with that.

Reply:
Ok, what do you think about `contextActionClicked`?

Reply:
Sounds good to me.